src/ui/components/IdentityListItems.js

```diff
@@ -1,24 +1,34 @@
 // @flow
 import React, {type Node as ReactNode} from "react";
-import {Divider, ListItem} from "@material-ui/core";
-import {type Identity} from "../../core/identity";
+import {Divider, ListItem, ListItemText, Checkbox} from "@material-ui/core";
+import {type Identity, type IdentityId} from "../../core/identity";
 
 type IdentityListItemsProps = {
   +identities: $ReadOnlyArray<Identity>,
   +onClick: Function,
+  +isCheckedHash: Map<IdentityId, boolean>,
+  +onCheckbox: Function,
```

Comment:
The type signature of just `Function` means that anyone can pass it any params and it won't complain, which isn't great for type safety. Its usage below indicates it should be a function that accepts an identityId as its first param and a change event as its second param.